packages/graphql/src/schema-model/generate-model.ts

```diff
@@ -121,8 +131,84 @@ function generateCompositeEntity(
     });
 }
 
+function hydrateRelationships(
+    definition: ObjectTypeDefinitionNode,
+    schema: Neo4jGraphQLSchemaModel,
+    definitionNodes: DefinitionNodes
+): void {
+    const name = definition.name.value;
+    const entity = schema.getEntity(name);
+
+    if (!(entity instanceof ConcreteEntity)) {
+        throw new Error(`Cannot add relationship to non-concrete entity ${entity.name}`);
+    }
+
+    const relationshipPropertyInterfaces = getRelationshipPropertiesInterfaces(definitionNodes);
+    if (!entity) throw new Error(`Entity ${name} not found while creating relationships`);
```

Comment:
I would raise this error before!

Reply:
Good catch, in fact, this error is not needed, as `getEntity` already throws if none is found